server/src/main/java/org/opensearch/gateway/PrimaryShardAllocator.java

```diff
@@ -457,6 +458,16 @@ private static NodesToAllocate buildNodesToAllocate(
 
     protected abstract FetchResult<NodeGatewayStartedShards> fetchData(ShardRouting shard, RoutingAllocation allocation);
 
+    @Override
+    // to be override
```

Comment:
Is the comment relevant, given the annotation ?

Reply:
will remove